typhos/utils.py

```diff
@@ -178,6 +193,17 @@ def __init__(self, *args, **kwargs):
         self._animation.setScaledSize(self._icon_size)
         self.setMovie(self._animation)
         self._animation.start()
+        if enable_timeout:
+            QtCore.QTimer.singleShot(self.LOADING_TIMEOUT_MS,
+                                     self._handle_timeout)
+
+    def _handle_timeout(self):
+        self._animation.stop()
+        self.setMovie(None)
+        self.setText("Loading Timeout")
+        self.setStyleSheet("color: red;")
```

Comment:
Should be in the stylesheet file

Reply:
Addressed via https://github.com/pcdshub/typhos/pull/331/commits/2e344b8798d9fc7cdd34c8190688a1271922fb7e